desktop/mac/release-notes/index.md

```diff
@@ -17,14 +17,45 @@ redirect_from:
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
-## Docker Desktop 4.1.1
-2021-10-12
+## Docker Desktop 4.2.0
+2021-11-xx
 
 > Download Docker Desktop
 >
 > [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
 > [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
 
+### New
+- **Pause/Resume**: Save your battery with Docker Pause / Resume! For more information, see [Docker Pause](https://docs.docker.com/go/desktop-pause/).
```

Comment:
The `/go/xxx` links are intended to be used for "in-product" links, but in the docs itself, we should use the direct link to the page/section.

Looks like this content will be added in https://github.com/docker/docker.github.io/pull/13776, so perhaps we should combine both PRs.

With https://github.com/docker/docker.github.io/pull/13776 in place, this should then become;

```suggestion
- **Pause/Resume**: Save your battery with Docker Pause / Resume! For more information, see [Docker Pause](../index.md#pauseresume).
```

Also wondering if we should use `Docker Pause` as caption for the link, as it may be confused with the [`docker pause` subcommand](https://docs.docker.com/engine/reference/commandline/pause/).

Perhaps "the pause/resume section in the user manual"

```suggestion
- **Pause/Resume**: Save your battery with Docker Pause / Resume! For more information, see the [pause/resume section](../index.md#pauseresume) in the user manual.
```

Reply:
**Pause/Resume**: You can now pause your Docker Desktop session when you are not actively using it and save CPU resources on your machine. For more information, see [Pause/Resume](../index.md#pauseresume).